spec/models/student_registration_spec.rb

```diff
@@ -3,8 +3,14 @@
 describe StudentRegistration do
   context 'when registering' do
 
-    it 'attempts to save the user'
-    it 'if the save is successful it creates empty project'
+    it 'if the save is successful it creates empty project' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.